packages/types/src/event-bus/event-bus.ts

```diff
@@ -0,0 +1,38 @@
+import { EmitData, Subscriber, SubscriberContext } from "."
```

Comment:
Maybe it is a good idea to avoid this circular reference between `index.ts` and `event-bus.ts`.
I suggest we create a `common.ts`, move all the types declared on `index.ts` there and `index.ts` only exports everything

Reply:
+1